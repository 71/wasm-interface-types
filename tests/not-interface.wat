;; parse-fail

x

(; CHECK-ALL:
expected `(`
     --> tests/not-interface.wat:3:1
      |
    3 | x
      | ^
;)
