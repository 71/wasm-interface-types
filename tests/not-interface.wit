;; parse-fail

x

(; CHECK-ALL:
  expected `(`
       --> tests/not-interface.wit:3:1
        |
      3 | x
        | ^

  Backtrace disabled; run with RUST_LIB_BACKTRACE=1 environment variable to display a backtrace
;)
