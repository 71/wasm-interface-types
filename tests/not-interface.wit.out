expected `(`
     --> ../../tests/not-interface.wit:3:1
      |
    3 | x
      | ^
